src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,8 +1,43 @@
 package core.basesyntax;
 
-public class WorkWithFile {
+public class WorkWithFile extends FileManager {
```

Comment:
```suggestion
public class WorkWithFile {
```
this class does not extend `FileManager`. if you want to use `FileManager`, create an instance where it is needed